ComposeAdvanced/app/src/main/AndroidManifest.xml

```diff
@@ -11,21 +10,20 @@
      See the License for the specific language governing permissions and
      limitations under the License.
 -->
-<manifest xmlns:tools="http://schemas.android.com/tools"
-    xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.WAKE_LOCK" />
 
     <uses-feature android:name="android.hardware.type.watch" />
 
     <application
+        android:name=".presentation.BaseApplication"
         android:allowBackup="true"
         android:fullBackupOnly="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:name=".presentation.BaseApplication"
-        android:supportsRtl="true"
-        android:theme="@style/SplashTheme">
```

Comment:
Does this remove it from maps?

Reply:
I don't think we ever showed splash screen theme, I actually reverted this bit to keep splash theme under application's attribute, not activities'